contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -950,6 +962,14 @@ pltsql_GetNewTempOidWithIndex(Relation relation, Oid indexId, AttrNumber oidcolu
 	{
 		CHECK_FOR_INTERRUPTS();
 
+		/*
+		* We do not expect a lot of collision, so this log should be fine
+		*/
+		if (retries > 0)
+		{
+			elog(NOTICE, "There is a collision with oid %u when determing oid for a temp object.", newOid);
```

Comment:
please include other information here such as the OID and/or name of the catalog. 

I am also not 100% sure if NOTICE is the right level here (and for the other messages), I believe this logs it to the client which is unnecessary and might expose too much underlying behavior. LOG may be better. (see comments in `elog.h`).